src/shared/components/radar/radar.component.tsx

```diff
@@ -26,7 +26,7 @@ export const Radar = ({
 }: RadarProps) => {
   const config: RadarConfig = {
     svg_id: 'radar',
-    width: zoomedQuadrant ? min(window.innerWidth - 360, BASIC_RADAR_WIDTH) : window.innerHeight + 210,
+    width: zoomedQuadrant ? min(window.innerWidth - 420, BASIC_RADAR_WIDTH) : window.innerHeight + 210,
```

Comment:
I think that those values like `420` and `210` might be added as constants as well. It would be easier to understand why they are there.